masonite/response.py

```diff
@@ -66,7 +66,7 @@ def converted_data(self):
         Returns:
             string -- Returns a string representation of the data
         """
-        if isinstance(self.data(), dict) or isinstance(self.data(), list):
+        if isinstance(self.data(), (dict,list)):
```

Comment:
Can you just add space after the comma
```suggestion
        if isinstance(self.data(), (dict, list)):
```

Reply:
sure.

Reply:
done